lib/vsc/filesystem/quota/tools.py

```diff
@@ -396,8 +396,8 @@ def process_inodes_information(filesets, quota, threshold=0.9):
     critical_filesets = dict()
 
     for (fs_key, fs_info) in filesets.items():
-        allocated = int(fs_info['allocInodes'])
-        maxinodes = int(fs_info['maxInodes'])
+        allocated = int(fs_info['allocInodes']) if storage == 'gpfs' else 0
+        maxinodes = int(fs_info['maxInodes']) if storage == 'gpfs' else int(quota[fs_key][0].filesLimit)
```

Comment:
Not a critique, but who the heck thought this was clean syntax? :p 

if only if/else statements delivered a value :-p